src/components/dialogs/network-modifications/line/modification/line-modification-dialog.jsx

```diff
@@ -223,6 +255,18 @@ const LineModificationDialog = ({
                 microUnitToUnit(characteristics[B2]),
                 currentLimits1,
                 currentLimits2,
+                connectivity1[VOLTAGE_LEVEL]?.id,
+                connectivity1[BUS_OR_BUSBAR_SECTION]?.id,
+                connectivity2[VOLTAGE_LEVEL]?.id,
+                connectivity2[BUS_OR_BUSBAR_SECTION]?.id,
+                sanitizeString(connectivity1[CONNECTION_NAME]),
+                sanitizeString(connectivity2[CONNECTION_NAME]),
+                connectivity1[CONNECTION_DIRECTION] ?? UNDEFINED_CONNECTION_DIRECTION,
+                connectivity2[CONNECTION_DIRECTION] ?? UNDEFINED_CONNECTION_DIRECTION,
+                connectivity1[CONNECTION_POSITION] ?? null,
+                connectivity2[CONNECTION_POSITION] ?? null,
+                connectivity1[CONNECTED] ?? null,
+                connectivity2[CONNECTED] ?? null,
```

Comment:
```suggestion
                connectivity1[CONNECTION_DIRECTION],
                connectivity2[CONNECTION_DIRECTION],
                connectivity1[CONNECTION_POSITION],
                connectivity2[CONNECTION_POSITION],
                connectivity1[CONNECTED],
                connectivity2[CONNECTED],
```
Why you need set default value for these fields?

Reply:
In addition, UNDEFINED_CONNECTION_DIRECTION as a default value is not coherent to null that you set in createConnectivityData